### PR TITLE
Adding support for AArch64

### PR DESCRIPTION
**Pull request by gunnarmorling**
@rmoff, this adds the native support for Arm to your pending PR. Note it requires https://github.com/decodableco/decodable/pull/3818 to be merged first and a release cut from it.

### CODE REVIEW

**Review by rmoff — APPROVED** [review]

File: cli-docker/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
-FROM amd64/ubuntu:latest
+FROM ubuntu:latest
+
+ARG TARGETARCH
 
 # Set environment variables
 ENV DECO_CLI_VERSION=1.20.0
```
Comment (gunnarmorling) [inline cli-docker/Dockerfile]:
```suggestion
ENV DECO_CLI_VERSION=1.21.0
```

**Comment by gunnarmorling** [comment]
@jbreeden, could you ping here once the Arm CLI release is out? Thx!

File: cli-docker/Dockerfile [hunk 2]
```diff
@@ -1,8 +1,10 @@
-FROM amd64/ubuntu:latest
+FROM ubuntu:latest
+
+ARG TARGETARCH
 
 # Set environment variables
-ENV DECO_CLI_VERSION=1.20.0
-ENV DECO_CLI_URL=https://releases.decodable.co/decodable-cli/linux/amd64/decodable-cli-linux-amd64-${DECO_CLI_VERSION}.tar.gz
+ENV DECO_CLI_VERSION=1.21.0
```
Comment (gunnarmorling) [inline cli-docker/Dockerfile]:
```suggestion
ENV DECO_CLI_VERSION=1.20.1
```

**Comment by rmoff** [comment]
I already merged cli-docker into main; do you want to do another PR for
this change?

-- 
Robin Moffatt
Principal DevEx Engineer, Decodable

**Comment by gunnarmorling** [comment]
Ah, whoops, good spot. Given you've approved this PR already, I've taken the liberty and just pushed this commit to main. Thx for bringing this up!

**Pull request merged**